python-oop-si-exercises-martikecskemeti/2-100-doors/main.py

```diff
@@ -17,16 +17,20 @@ def generate_doors(count=100):
 #   - doors: list of Door objects
 # Return: list of toggled Door instances
 def toggle_doors(doors):
-    return []
+    for i in range(1, len(doors) + 1):
+        for j in range(i, len(doors) + 1, i):
+            if doors[j - 1].id == j:
```

Comment:
Ez igy nem igazan konzisztens. Ket lehetoseg van, vagy feltetelezed, hogy sorba jonnek `id` szerint, vagy nem. Ha feltetelezed, akkor nem kell az `if`. Ha nem feltetelezed, akkor meg kellene keresned azt az indexet-t, ahol `doors.id == j`. A legbiztosabb megoldas az lenne, ha beraknad az osszes door-t egy dict-be `id` szerint.